scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -470,17 +491,21 @@ def validate_model_assets(latest_asset_config: assets.AssetConfig, validated_ass
             if latest_model_config.type != validated_model_config.type:
                 logger.log_warning(f"latest_model_config_type: [{latest_model_config.type}]")
                 logger.log_warning(f"validated_model_config_type: [{validated_model_config.type}]")

```

Comment:
Would it be possible to make clear that the type does not match in the warning message (similar to what you did with the description warning in which you said, "Description does not match, for latest and validated asset"). For example, maybe something like: 

```python
logger.log_warning(f"latest_model_config_type: {latest_model_config.type} does not match validated_model_config_type: {validated_model_config.type}")
```

Reply:
Thanks, updated